indexer/index.py

```diff
@@ -438,7 +401,7 @@ def index(
     parser.add_argument(
         "-n",
         "--non-recursive",
-        default=False,
+        default=defaults.NON_RECURSIVE,
         action="store_true",
         help="do not recursively index / do not descend into subdirectories",
     )
```

Comment:
Good catch. I didn't realize that I had loaded that foot gun.

I'll keep the defaults as `False` and just remove the constants altogether. Considering the way these things propagate down from the command line, it'll be awfully confusing if we do decide to update the constant and nothing seems to be different because we've been using them in a way that ignores them anyway.